internal/pkg/cli/svc_init.go

```diff
@@ -430,7 +438,22 @@ If you'd prefer a new default manifest, please manually delete the existing one.
 }
 
 func (o *initSvcOpts) askStaticSite() error {
-	// TODO: add file selection for generating svc manifest.
+	var assets []manifest.FileUpload
+	sources, err := o.askSource()
```

Comment:
ohhh we should have a way to skip the prompt!

Reply:
we do! it's in the next PR that's waiting in the wings. :)